js/web/lib/wasm/jsep/webgpu/ops/attention.ts

```diff
@@ -246,9 +246,7 @@ const createInPlaceSoftmaxProgramInfo = (_context: ComputeContext, input: Tensor
   let WG = 64;
   const dComp = d / components;
   if (dComp < WG) {
-    WG = 1;
-  } else if (dComp / 8 < 64) {
-    WG = Math.ceil(dComp / 8);
+    WG = 32;
```

Comment:
It's not necessary to set it to a smaller value since in hardware it at least has the SIMD width threads to execute simultaneously.